mslice/models/slice/slice_cache.py

```diff
@@ -0,0 +1,63 @@
+from mslice.models.slice.slice_functions import (compute_chi, compute_chi_magnetic, compute_d2sigma, compute_gdos,
+                                                 compute_symmetrised)
+
+
+class SliceCache():
+    '''class that caches intensities and metadata for a single workspace'''
```

Comment:
This class seems to contain all the important data related to a slice, is it worth increasing the documentation to make clear what each of the parameters in here are? This is probably documented elsewhere and if so ignore this comment.